Makefile

```
@@ -1,5 +1,4 @@
-all         :; DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=200 dapp --use solc:0.8.14 build
-clean       :; dapp clean
+all         :; forge build --use solc:0.8.14
```

```suggestion
all         :; FOUNDRY_OPTIMIZER=true FOUNDRY_OPTIMIZER_RUNS=200 forge build --use solc:0.8.14
```
Even if it is the default might make sense to be very explicit.